frontend/src/Components/Account/Register.js

```diff
@@ -1,55 +1,106 @@
 import React, { useState } from "react";
 import { navigate, A } from "hookrouter";
 import axios from "axios";
+import { validateEmailAddress, validatePassword } from "../Utils/validations";
 
 export default function Register() {
   const initForm = {
     name: "",
     email: "",
     password: "",
-    profile_picture: null,
+    confirm_password: "",
     description: "",
   };
-  //Confirm Password in BE and FE
+
+  const initError = {
+    name: "",
+    email: "",
+    password: "",
+    confirm_password: "",
+    description: "",
+  };
+
   const [form, setForm] = useState(initForm);
   const [formError, setFormError] = useState(false);
   const [fileInterface, setFile] = useState({ fileUpload: null });
+  const [error, setError] = useState(initError);
+
+  function validInputs() {
+    let formValid = true;
+    let err = Object.assign({}, initError);
+    const { password, confirm_password, email } = form;
+
+    Object.keys(form).forEach((key) => {
+      if (key != "description" && form[key] === "") {
+        formValid = false;
+        err[key] = "This field is required";
+      }
+    });
+
+    if (password !== confirm_password) {
+      err["confirm_password"] = "Passwords do not match";
+      formValid = false;
+    }
```

Comment:
Since we're already doing this here, do we need to do this in the backend as well?

Reply:
client-side validation can be totally bypassed by using curl commands. We use validation here to just provide the user instant response without even sending the request and server-side to be sure that the input is actually validated